src/Silakov.Homework4/StatusTask.cs

```diff
@@ -0,0 +1,25 @@
+﻿namespace Silakov.Homework4
+{
+    /// <summary>
+    /// Статусы задач.
+    /// </summary>
+    internal enum StatusTask
+    {
+        /// <summary>
+        /// Запланирована.
+        /// </summary>
+        planned = 1,
```

Comment:
let's use a common naming convention for enums where member names start from upper case character